MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -805,6 +813,23 @@ private void BuildStreamVideoItem(StreamInfo playlistItem, VideoOptions options,
 
             // Prefer matching audio codecs, could do better here
             var audioCodecs = ContainerProfile.SplitValue(audioCodec);
+
+            // Enforce HLS audio codec restrictions
+            if (string.Equals(playlistItem.SubProtocol, "hls", StringComparison.OrdinalIgnoreCase))
+            {
+                var supportedHlsAudioCodecs = new List<string> { "aac", "ac3", "eac3", "mp3" };
+                if (string.Equals(playlistItem.Container, "mp4", StringComparison.OrdinalIgnoreCase))
+                {
+                    // fMP4 supports more codecs than TS
+                    supportedHlsAudioCodecs.Add("alac");
+                    supportedHlsAudioCodecs.Add("flac");
+                    supportedHlsAudioCodecs.Add("opus");
+                    supportedHlsAudioCodecs.Add("vorbis");
```

Comment:
We currently do not support vorbis in HLS.

https://github.com/jellyfin/jellyfin/blob/edc627fd5b5a5bd19c843dd9e2970b1ebce3fbfd/Jellyfin.Api/Helpers/DynamicHlsHelper.cs#L598-L637